lib/disco_log/presence.ex

```diff
@@ -71,14 +71,19 @@ defmodule DiscoLog.Presence do
   # https://discord.com/developers/docs/events/gateway#identifying
   def handle_continue(
         :identify,
-        %__MODULE__{discord_config: config, websocket_client: client} = state
+        %__MODULE__{
+          discord_config: config,
+          websocket_client: client,
+          presence_status: presence_status
+        } = state
       ) do
     identify_event = %{
       op: 2,
       d: %{
         token: config.token,
         intents: 0,
-        presence?: %{
+        presence: %{
+          activities: [%{type: 4, state: presence_status, name: "Name"}],
```

Comment:
What's the `name: "Name"`
Does it show somewhere ?

Reply:
It's a required field in [presence](https://discord.com/developers/docs/events/gateway-events#activity-object) object. It's shown in a lot of presence type, but for custom type to my knowledge it's not used. The `state` is displayed instead